synAnnotationUtils/get-geo-annotations.R

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env Rscript
+
+suppressPackageStartupMessages(library("synapseClient"))
+suppressPackageStartupMessages(library("GEOquery"))
+suppressPackageStartupMessages(library("SRAdb"))
+suppressPackageStartupMessages(library("plyr"))
+suppressPackageStartupMessages(library("optparse"))
+
+option_list <- list(
+                    make_option(c("--gse"), action="store",
+                                default=NULL,
+                                help="GEO Series accession number (e.g., \"GSE89777\")"),
+                    make_option(c("--output-file"), action="store",
```

Comment:
Consider just writing to stdout and not supplying a file name (then use command line redirect to capture). Standard mechanism for command line programs.